components/brave_wallet_ui/components/extension/connect-with-site-panel/select-account-item/select-account-item.tsx

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import { create } from 'ethereum-blockies'
+
+// Types
+import {
+  WalletAccountType,
+  SupportedTestNetworks
+} from '../../../../constants/types'
+
+// Selectors
+import { WalletSelectors } from '../../../../common/selectors'
+import { useUnsafeWalletSelector } from '../../../../common/hooks/use-safe-selector'
+
+// Styled Components
+import {
+  ConnectPanelButton,
+  AccountAddressText,
+  AccountNameText,
+  BalanceText,
+  NameAndAddressColumn,
+  AccountCircle,
+  LeftSide,
+  SelectedIcon
+} from './select-account-item.style'
+import { LoadingSkeleton } from '../../../shared'
+
+// Utils
+import { reduceAccountDisplayName } from '../../../../utils/reduce-account-name'
+import { reduceAddress } from '../../../../utils/reduce-address'
+import { computeFiatAmount } from '../../../../utils/pricing-utils'
+import { getBalance } from '../../../../utils/balance-utils'
+import Amount from '../../../../utils/amount'
+
+interface Props {
+  account: WalletAccountType
+  isSelected: boolean
+  onSelectAccount: () => void
+}
+
+export const SelectAccountItem = (props: Props) => {
+  const { account, isSelected, onSelectAccount } = props
+
+  // Wallet Selectors
+  const networks = useUnsafeWalletSelector(WalletSelectors.networkList)
+  const selectedNetwork = useUnsafeWalletSelector(
+    WalletSelectors.selectedNetwork
+  )
+  const userVisibleTokensInfo = useUnsafeWalletSelector(
+    WalletSelectors.userVisibleTokensInfo
+  )
+  const spotPrices = useUnsafeWalletSelector(
+    WalletSelectors.transactionSpotPrices
+  )
+  const defaultCurrencies = useUnsafeWalletSelector(
+    WalletSelectors.defaultCurrencies
+  )
```

Comment:
```suggestion
  const defaultFiatCurrency = useUnsafeWalletSelector(
    WalletSelectors.defaultFiatCurrency
  )
```

only the default fiat currency was used

Reply:
Updated!